docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -98,6 +98,188 @@ Interventions
+
+	Add descriptions of strategy and assumptions regarding differences between baseline and intervention coverage (intervention effect lag, non-age-dependent, SOURCES for these)
+
+**Vivarium Modeling Strategy**
+
+In our Vivarium simulation, the effect of exposure foods **not** fortified 
+with vitamin A on the prevalence of vitamin A deficiency realtive to those 
+exposed to vitamin A fortified foods will be represented as follows: 
+
+.. math::
+
+  RR = \frac{P(\text{VAD prevalence} \mid \text{no fortification})}
+  {P(\text{VAD prevalence} \mid \text{fortification})}
+  \approx \frac{1}{0.45\: (0.19, 1.05)}
+  \approx 2.22\: (0.95, 5.26).
+
+.. note::
+
+	We are modeling the reciprocal of the relative risk reported in the 
+	Cochrane review.
+
+	Additionally, this effect size crosses the null, and therefore, in some 
+	draws it will cause increasing coverage of the intervention to *increase* 
+	vitamin A deficiency prevalence. This is a limitation caused by the low 
+	quality evidence regarding the relative risk of vitamin A fortification on 
+	vitamin A prevalence. However, on average, increasing coverage of vitamin 
+	A fortification will decrease VAD.
+
+To model the uncertainty in this estimate, the above RR should be drawn from a
+`lognormal <https://en.wikipedia.org/wiki/Log-normal_distribution>`_
+distribution with median = 2.22, 2.5\ :superscript:`th`-percentile = 0.95, and
+97.5\ :superscript:`th`-percentile = 5.26. This distbibution can be created
+using `SciPy's lognorm function
+<https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.lognorm.html>`_
+as follows:
+
+.. code-block:: Python
+
+  from numpy import log
+  from scipy.stats import norm, lognorm
+
+  # median and 0.975-quantile of lognormal distribution for RR
+  median = 2.22
+  q_975 = 5.26
+
+  # 0.975-quantile of standard normal distribution (=1.96, approximately)
+  q_975_stdnorm = norm().ppf(0.975)
+
+  mu = log(median) # mean of normal distribution for log(RR)
+  sigma = (log(q_975) - mu) / q_975_stdnorm # std dev of normal distribution for log(RR)
+
+  # Frozen lognormal distribution for RR, representing uncertainty in our effect size
+  # (s is the shape parameter)
+  rr_distribution = lognorm(s=sigma, scale=median)
+
+.. note::
+
+	I copied this from Nathaniel's documentation for the folic acid RR, but I 
+	think that the same approach is appropriate. Perhaps we can eventually 
+	create a separate page that lists similar strategies that we can reference 
+	via links.
+
+Additionally, as described in the research considerations above, the 
+intervention effect is dependent on age and time since intervention coverage.
+
+For simulants covered by *baseline coverage*, the effect of the intervention 
+is applied continuously from the start of the simulation to the end of the 
+simulation and only depends on age, such that:
+
+.. code-block:: Python
+
+  if age_i < 0.5:
+  	rr_i = 1 + (rr - 1) * a_i / 0.5
+  else:
+  	rr_i = rr
+
```

Comment:
Two comments:
1. Make sure the variable names are consistent, i.e. should a_i be age_i?
2. It would be good to explicitly say what the variable rr is before you use it. Continuing the code from where the lognormal distribution was defined, you could precede this if/else statement by rr = rr_distribution.rvs() to generate a sample value of the relative risk and store it in the variable rr.

Reply:
updated locally